pkg/resource/deploy/step_generator.go

```diff
@@ -1242,16 +1276,40 @@ func (sg *stepGenerator) calculateDependentReplacements(root *resource.State) ([
 
 func (sg *stepGenerator) AnalyzeResources() result.Result {
 	resourcesSeen := sg.resourceStates
-	resources := make([]plugin.AnalyzerResource, 0, len(resourcesSeen))
-	for _, v := range resourcesSeen {
-		resources = append(resources, plugin.AnalyzerResource{
-			URN:  v.URN,
-			Type: v.Type,
-			Name: v.URN.Name(),
-			// Unlike Analyze, AnalyzeStack is called on the final outputs of each resource,
-			// to verify the final stack is in a compliant state.
-			Properties: v.Outputs,
-		})
+	resources := make([]plugin.AnalyzerStackResource, 0, len(resourcesSeen))
+	for urn, v := range resourcesSeen {
+		goal := sg.resourceGoals[urn]
+		resource := plugin.AnalyzerStackResource{
+			AnalyzerResource: plugin.AnalyzerResource{
+				URN:  v.URN,
+				Type: v.Type,
+				Name: v.URN.Name(),
+				// Unlike Analyze, AnalyzeStack is called on the final outputs of each resource,
+				// to verify the final stack is in a compliant state.
```

Comment:
This isn't new - but I don't understand this comment in this context.

Reply:
I think the comment is trying to point out that `AnalyzeStack` passes the outputs, rather than passing the inputs like `Analyze`.